Sources/SectionLabel.swift

```diff
@@ -1,16 +1,10 @@
-//
-//  SectionLabel.swift
-//  Demo
-//
-//  Created by Marijn Schilling on 06/06/16.
-//
-//
-
 import UIKit
 
 class SectionLabel: UIView {
 
-    static let RightOffsetForSectionLabel : CGFloat = 90.0
+    static let RightOffsetForActiveSectionLabel: CGFloat = 80.0
+    static let RightOffsetForInactiveSectionLabel: CGFloat = 60.0
+
```

Comment:
Hmm, I'm assuming the app is iPhone only, so how does this look on the 3 different iPhone sizes? Do they all look fine with the same distance? 🤔